saleor/graphql/checkout/mutations/checkout_delivery_method_update.py

```diff
@@ -61,6 +62,16 @@ class Meta:
         )
         doc_category = DOC_CATEGORY_CHECKOUT
         error_type_class = CheckoutError
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventSyncType.SHIPPING_LIST_METHODS_FOR_CHECKOUT,
+                description="Triggered when provided delivery method is external id.",
```

Comment:
Rephrase for clarity: "Triggered when provided delivery method is an external one, provided by this webhook."